pkg/build/builder/docker.go

```diff
@@ -210,6 +214,11 @@ func (d *DockerBuilder) addBuildParameters(dir string) error {
 	instructions := dockerfile.ParseTreeToDockerfile(node)
 
 	// Overwrite the Dockerfile.
+	f, err := os.Open(dockerfilePath)
```

Comment:
the `fi.Mode()` of this file will be used in the last line of the method, any way to get the fileMode without open the file ?